_docs-sources/intro/overview/josh-feedback.md

```diff
@@ -0,0 +1,66 @@
+---
+'title': "Josh's Feedback"
+---
+
+# Josh's Feedback
+
+- What's the vision for "tool fundamentals"?
+  - How does it compare to Set Up Your Environment / Installing Neccessary Tools?
+  - Should "Setup Your Tools" (formerly Tool Fundamentals) be part of Set Up Your Environment?
```

Comment:
The vision for "set up your environment is that you could use e.g. docker or brew to bootstrap your whole setup quickly (although manual install would also be documented). The tools section would actually introduce the tools and their concepts. This may not be a useful distinction depending on how much setup the tools require beyond installing them.